09-gis.Rmd

```diff
@@ -618,87 +618,82 @@ In addition, if you would like to run simulations with the help of a geodatabase
 
 Please note that there are a number of further GIS software packages that have a scripting interface but for which there is no dedicated R package that accesses these: gvSig, OpenJump, Orfeo Toolbox and TauDEM.
 
-## A short note on further spatial APIs
+## Other bridges
```

Comment:
yeah, why not, let's call it other bridges (though nobody has done so before, usually people say API)

Reply:
The chapter's called bridges and the term's already defined. Deliberately re-define APIs early in case people have forgotten since chapter 1 or when it was last mentioned.